nntrainer/layers/activation_layer.h

```diff
@@ -85,6 +86,11 @@ class ActivationLayer : public Layer {
    */
   const std::string getType() const override { return ActivationLayer::type; };
 
+  /**
+   * @copydoc Layer::supportInPlace()
+   */
+  bool supportInPlace() const override { return support_in_place; }
```

Comment:
how about `acti_type != ActivationType::ACT_SOFTMAX` instead of member variable?

Reply:
it works, for now, might create problems when more activation types are added.

Reply:
Yes, I was thinking it'd better have to calculate right on the way because this guarantees invariant at the cost of calculation, which in this case looks OK deal.

It's just a suggestion leaving decision up to you :)

Reply:
updated. Also, moved decision making to ActiFunc.